rts/at/tx/src/main/java/org/jboss/jbossts/star/resource/Transaction.java

```diff
@@ -234,7 +234,7 @@ public boolean isRunning()
     }
 
 
-    private RESTRecord findParticipant(String participantUrl) {
+    protected RESTRecord findParticipant(String participantUrl) {
```

Comment:
I don't think this needed to be protected?

Reply:
I have backed out all changes to this file.